aws-env.dev

```diff
@@ -0,0 +1,8 @@
+## This file store all environment variables for the DEV environment
+PROJECT = lambda2sqs
+AWS_REGION = ap-southeast-1
+DEPLOY_S3_PREFIX = lambda2sqs
+AWS_PROFILE = uneet-dev
```

Comment:
@tuananh1508 if I'm not mistaken you should remove the spaces 
`PROJECT = lambda2sqs` --> `PROJECT=lambda2sqs`
`AWS_REGION = ap-southeast-1` --> `AWS_REGION=ap-southeast-1`

etc...